src/components/10-atoms/heading/story.js

```diff
@@ -0,0 +1,37 @@
+/* global document */
+import { storiesOf } from '@storybook/html';
+// if your need more boolean, select, radios
+import { withKnobs } from '@storybook/addon-knobs';
+import { html, render } from 'lit-html';
+import './index';
+import Readme from './README.md';
+
+const storyAXAHeading = storiesOf('Atoms/Heading', module);
+storyAXAHeading.addDecorator(withKnobs);
+storyAXAHeading.addParameters({
+  readme: {
+    sidebar: Readme,
+  },
+});
+
+storyAXAHeading.add('Heading', () => {
+  const wrapper = document.createElement('div');
+  const template = html`
+    <axa-heading rank="1">H1 Primary Heading</axa-heading>
+    <axa-heading rank="2">H2 Primary Heading</axa-heading>
+    <axa-heading rank="3">H3 Primary Heading</axa-heading>
+    <axa-heading rank="4">H4 Primary Heading</axa-heading>
+    <axa-heading rank="5">H5 Primary Heading</axa-heading>
+    <axa-heading rank="6">H6 Primary Heading</axa-heading>
+
+    <axa-heading variant="secondary" rank="1">H1 Secondary Heading</axa-heading>
+    <axa-heading variant="secondary" rank="2">H2 Secondary Heading</axa-heading>
+    <axa-heading variant="secondary" rank="3">H3 Secondary Heading</axa-heading>
+    <axa-heading variant="secondary" rank="4">H4 Secondary Heading</axa-heading>
+    <axa-heading variant="secondary" rank="5">H5 Secondary Heading</axa-heading>
+    <axa-heading variant="secondary" rank="6">H6 Secondary Heading</axa-heading>
```

Comment:
I would suggest using a loop?

Reply:
Imho this is more readable, since it's a declarative approach. More opinions on this are welcome though. Good input!

This would be the result. Unreadable and doesn't save any lines. Could be optimized, but If this doesn't become less than three lines, it's imho not worth the effort.

```
storyAXAHeading.add('Heading', () => {
  const wrapper = document.createElement('div');

  const template = [];
  for (let i = 1; i < 7; ++i) {
    template.push(html`
      <axa-heading rank="${i}">H${i} Primary Heading</axa-heading>
    `);
  }
  for (let i = 1; i < 7; ++i) {
    template.push(html`
      <axa-heading rank="${i}" variant="secondary"
        >H${i} Primary Heading</axa-heading
      >
    `);
  }

  render(template, wrapper);
  return wrapper;
});
```

Reply:
You are right. It's definitely more readable.